Appraisals

```diff
@@ -20,28 +15,40 @@ appraise "rails5.1" do
 end
 
 appraise "rails5.2" do
-  gem "activerecord", "~> 5.2.3"
-  gem "railties", "~> 5.2.3"
-  gem "actionmailer", "~> 5.2.3"
-  gem "bootsnap"
-  gem "activestorage", "~> 5.2.3"
-  gem "sass-rails"
+  gem "bootsnap", ">= 1.1.0", require: false
+  gem "byebug", platforms: :ruby
+  gem "capybara", ">= 2.15"
+  gem "chromedriver-helper"
+  gem "coffee-rails", "~> 4.2"
+  gem "jbuilder", "~> 2.5"
+  gem "puma", "~> 3.11"
+  gem "rails", "~> 5.2.4", ">= 5.2.4.2"
+  gem "sass-rails", "~> 5.0"
+  gem "selenium-webdriver"
+  gem "sqlite3", "~> 1.3.6", platforms: :ruby
+  gem "turbolinks", "~> 5"
+  gem "uglifier", ">= 1.3.0"
+  gem "listen", ">= 3.0.5", "< 3.2"
+  gem "spring"
+  gem "spring-watcher-listen", "~> 2.0.0"
+  gem "web-console", ">= 3.3.0", group: :development
 end
 
 appraise "rails6.0" do
-  gem "rails", "~> 6.0.0"
-  gem "bootsnap"
-  gem "byebug"
-  gem "capybara"
-  gem "jbuilder"
-  gem "listen", "~> 3.1.5"
-  gem "puma", "~> 3.11"
-  gem "sass-rails", "~> 5"
+  gem "bootsnap", ">= 1.4.2", require: false
+  gem "byebug", platforms: :ruby
+  gem "capybara", ">= 2.15"
+  gem "jbuilder", "~> 2.7"
+  gem "puma", "~> 4.1"
+  gem "rails", "~> 6.0.2", ">= 6.0.2.2"
```

Comment:
Regarding [this](https://github.com/cucumber/cucumber-rails/issues/461) issue, looks like it will failing for a while 😞

Reply:
😢 Thanks for looking into this. I'm going to see if I can cucumber to log more details about what is going wrong here.